samples/backchannel/index.html

```diff
@@ -65,7 +65,7 @@ <h2 style="font-family:Segoe UI;">Type a color into the WebChat!</h2>
                 secret: params['s'],
                 token: params['t'],
                 domain: params['domain'],
-                webSocket: params['webSocket'] // defaults to true
+                webSocket: params['webSocket'] && params['webSocket'] === "true" // defaults to true
```

Comment:
No because I want to pass in `undefined` in the (common) case where the `webSocket` parameter doesn't exist.